sdks/python/apache_beam/io/iobase.py

```diff
@@ -910,6 +913,67 @@ def from_runner_api_parameter(parameter, context):
     Read.from_runner_api_parameter)
 
 
+class SDFBoundedSourceWrapper(ptransform.PTransform):
```

Comment:
I don't believe we want to make this class public. It should be an implementation detail of how a runner is performing the replacement with eventually the Read PTransform performing the replacement.